### PR TITLE
Feature/custom assert

### PR DESCRIPTION
**Pull request by c3riley**
Added support for server side events
Fix name filter
add support for option method call
fix checkstyle

### CODE REVIEW

File: testah-junit/src/main/java/org/testah/driver/http/eventstream/ConsumeEventStream.java [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.ClientBuilder;
+import javax.ws.rs.client.WebTarget;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class ConsumeEventStream {
+
+    private final static String TIMEOUT_MSG = "SseFeature Took to long, went over 120 max";
```
Comment (gweiske) [inline testah-junit/src/main/java/org/testah/driver/http/eventstream/ConsumeEventStream.java]: What is SseFeature? Maybe something more informative here?

File: testah-junit/src/test/java/org/testah/MessageDto.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
+package org.testah;
+
+public class MessageDto {
+
+    private String type;
+    private String message;
+
+    public MessageDto() {
+
```
Comment (gweiske) [inline testah-junit/src/test/java/org/testah/MessageDto.java]: No need for non-trivial constructor?

File: testah-junit/src/test/java/org/testah/OptionCall.java [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+import org.junit.Test;
+import org.testah.driver.http.requests.OptionRequestDto;
+
+public class OptionCall {
+
+    @Test
+    public void test() throws Exception {
+
+        TS.http().addCustomTestHeader("option-test").doRequest(new OptionRequestDto("http://www.testah.com"));
```
Comment (gweiske) [inline testah-junit/src/test/java/org/testah/OptionCall.java]: Nothing to assert?

File: testah-junit/src/test/java/org/testah/http/OptionCall.java [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+import org.junit.Test;
+import org.testah.TS;
+import org.testah.driver.http.requests.OptionRequestDto;
+
+public class OptionCall {
+
+    @Test
+    public void test() throws Exception {
+
+        TS.http().addCustomTestHeader("option-test").doRequest(new OptionRequestDto("http://www.testah.com"));
```
Comment (gweiske) [inline testah-junit/src/test/java/org/testah/http/OptionCall.java]: Nothing to assert?

File: testah-junit/src/test/java/org/testah/MessageDto.java (hunk 2 above)
Comment (c3riley) [reply testah-junit/src/test/java/org/testah/MessageDto.java]: I usually always include a base constructor, older jackson used to need it.

**Pull request merged**